translate/cloud-client/beta_snippets.py

```diff
@@ -269,8 +269,8 @@ def translate_text_with_glossary(project_id, glossary_id, text):
         target_language_code='es',
         glossary_config=glossary_config)
 
-    for translation in result.translations:
-        print(translation)
+    for glossary_translation in result.glossary_translations:
```

Comment:
You will need to update the tests, I'm seeing test_translate_text failing.

Reply:
I fixed the tests!